packages/main/src/TabInStrip.hbs

```diff
@@ -64,5 +64,8 @@
 {{#*inline additionalText}}
 	{{#if this.additionalText}}
 		<span class="ui5-tab-strip-itemAdditionalText" id="{{this._id}}-additionalText">{{this.additionalText}}</span>
+
+	{{else if this.icon}}
+		<span class="ui5-tab-strip-itemAdditionalText-hidden" id="{{this._id}}-additionalText-hidden"></span>

```

Comment:
Refactor this to use a getter `classes` in Tab.ts like it is used on line 2. The object this getter will return would be accessed with `stripClasses.additionalText` which will contain the appropriate class `ui5-tab-strip-itemAdditionalText`, and include `ui5-tab-strip-itemAdditionalText--hidden`. Combine it with the existing getter `stripClasses` but change the type of the return value.

Reply:
Done